src/components/ElvaYBPractice/styles/RightComponent.styled.ts

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components'
+
+export const RightComponent = styled.div`
+  background-color: hsl(179, 62%, 48%);
+  width: 50%;
+  height: 45vh;
+  @media only screen and (max-width: 600px) {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    height: 30vh;
```

Comment:
Consider changing 30vh to auto so that content won't get cut off during mobile view